src/rf/js/src/user/models.js

```diff
@@ -44,6 +44,10 @@ var UserModel = Backbone.Model.extend({
         });
 
         return jqXHR;
+    },
+
+    getCreateLayerURL: function() {
+        return '/user/' + this.get('username') + '/layer/create';
```

Comment:
I was thinking about moving this to the user model JSON so consumers of the API don't have to reverse-engineer our code to figure out how to create a layer. We can do this later.